packages/block-library/src/accordion-content/edit.js

```diff
@@ -74,6 +73,10 @@ export default function Edit( {
 			className: clsx( blockProps.className, {
 				'is-open': openByDefault || isSelected,
 			} ),
+			style: {
+				...blockProps.style,
+				...spacingProps.style,
+			},
```

Comment:
This approach is only necessary if we want to skip serialization of block support or apply styles to elements within a block.

We might take this opportunity to simplify it to:

```javascript
const blockProps = useBlockProps( {
	className: clsx( {
		'is-open': openByDefault || isSelected,
	} ),
} );
const innerBlocksProps = useInnerBlocksProps( blockProps, {
	template: [
		[ 'core/accordion-header', {} ],
		[
			'core/accordion-panel',
			{
				openByDefault,
			},
		],
	],
	templateLock: 'all',
	directInsert: true,
	templateInsertUpdatesSelection: true,
} );
```